### ISSUE TITLE
3.0.0 does not work

### ISSUE THREAD

**Issue by steklopod** [nuxt-alt/auth#81]
### Environment

- Operating System: Linux
- Node Version:     v18.18.0
- Nuxt Version:     3.8.2
- CLI Version:      3.10.0
- Nitro Version:    2.8.0
- Package Manager:  npm@9.4.2
- Builder:          -
- User Config:      modules, auth
- Runtime Modules:  @nuxt-alt/auth@3.0.0, @pinia/nuxt@0.5.1
- Build Modules:    -

### Nuxt Config

```ts
import { defineNuxtConfig } from 'nuxt/config'
const host = 'http://localhost:8080'

export default defineNuxtConfig({
  modules: ['@nuxt-alt/auth', '@pinia/nuxt'],
  auth: {
    scopeKey: 'scope',
    globalMiddleware: true,
    routerStrategy: 'navigateTo',
    strategies: {
      localStorage: false,
      refresh: {
        scheme: 'refresh',
        provider: () => {}, // https://github.com/nuxt-alt/auth/issues/54#issuecomment-1574969850
      },
    },
  },
})
```

### Reproduction

https://stackblitz.com/edit/github-eptmme?file=nuxt.config.ts

### Describe the bug

<img width="1430" alt="Снимок экрана 2023-11-25 в 16 16 58" src="https://github.com/nuxt-alt/auth/assets/29500153/0e4b15f4-4e70-499e-85c6-c1774723d4b2">

<img width="1089" alt="Снимок экрана 2023-11-25 в 16 17 13" src="https://github.com/nuxt-alt/auth/assets/29500153/8867a7b3-fb4a-4f99-8894-e7475657ca7a">

### Additional context

Also it does not work on mac

### Logs

```shell
> watch.plugin.mjs?v=b05b8dee:1 Uncaught ReferenceError: defineNuxtPlugin is not defined
    at watch.plugin.mjs?v=b05b8dee:1:1
```

**Comment by steklopod** [comment]
Also in `release notes`:
<img width="933" alt="Снимок экрана 2023-11-25 в 16 20 03" src="https://github.com/nuxt-alt/auth/assets/29500153/210e9d1e-a9c7-4bbc-be76-544226207a52">

It can not start without pinia.

**Comment by Denoder** [comment]
@steklopod for now you will have to disable watchLoggedIn, it seems that it's registering the watch plugin before the auth plugin and it's utils.

Does this happen locally or only in stackblitz?

**Comment by toniengelhardt** [comment]
@Denoder I also see this locally. Will it work correctly if I disable `watchLoggedIn`?

**Comment by Denoder** [comment]
yes. If you need the watch plugin, just create one manually and register it til I release the next version.

**Comment by steklopod** [comment]
Thabks for quick reply 👍🏻

I tested with `watchLoggedIn: false,` inside `nuxt.config.ts` and it works.

What is it for? 

Can it be fixed in 3.0.1?

**Comment by toniengelhardt** [comment]
> yes. If you need the watch plugin, just create one manually and register it til I release the next version.

Thanks 🙏🏽
